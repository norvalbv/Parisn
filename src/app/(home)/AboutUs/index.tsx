import React, { ReactElement } from 'react';
import Radial from '@/src/components/SVG/Design';
import DASHBOARD_IMAGE from '@/public/images/dashboard-image.jpg';
const AboutUs = (): ReactElement => {
  return (
    <div className="relative mx-auto mb-20 flex w-full flex-col items-center gap-8 md:mb-40 md:w-11/12 md:flex-row md:gap-16 lg:w-9/12">
      <Radial className="right-1/4 hidden md:block" />
      <div className="w-full md:w-1/2">
        <span className="mb-4 inline-block text-base font-semibold uppercase tracking-[.42rem] md:mb-8 md:text-lg">
          Parisn.com
        </span>
        <p className="text-sm md:text-base">
          PARISN revolutionizes the traditional retail experience by introducing an innovative
          pricing mechanism. With us, you are empowered to decide the price you pay for your items.
          Every piece in our exclusive collection starts at a premium, reflecting its rarity and
          quality. However, as time progresses, the price of each item gracefully descends,
          eventually reaching the unique possibility of being free. This distinctive model disrupts
          the norms of conventional shopping, turning every purchase into a thrilling strategic
          decision and transforming every customer into a savvy shopper
        </p>
      </div>
      <div className="h-full w-full opacity-0 md:w-1/2">
        <img src={DASHBOARD_IMAGE.src} alt="p" className="h-auto w-full" />
      </div>
    </div>
  );
};

export default AboutUs;
