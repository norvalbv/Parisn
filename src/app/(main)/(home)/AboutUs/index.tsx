import DASHBOARD_IMAGE from '@/public/images/dashboard-image.jpg';
import Header from '@/src/components/Header';
import { COMPANY_NAME } from '@/src/constants';
import React, { ReactElement } from 'react';

const AboutUs = (): ReactElement => {
  return (
    <div className="relative mx-auto px-4 sm:px-6 lg:px-8 py-12 sm:py-16 md:py-20 max-w-7xl">
      <div className="flex flex-col md:flex-row items-center gap-8 sm:gap-12 md:gap-16">
        <div className="w-full md:w-1/2 space-y-4 sm:space-y-6">
          <Header title={COMPANY_NAME} className="text-center md:text-left" />
          <p className="text-sm sm:text-base lg:text-lg text-center md:text-left leading-relaxed sm:leading-relaxed">
            {COMPANY_NAME} brings a fresh approach to luxury retail. Our unique pricing model starts
            items at premium rates that gradually decrease over time. This innovative system lets you
            choose when to buy - whether at initial pricing or after reductions. It's shopping
            re-imagined, where patience could lead to significant savings.
          </p>
        </div>
        <div className="w-full md:w-1/2">
          <div className="relative rounded-lg overflow-hidden shadow-xl">
            <img
              src={DASHBOARD_IMAGE.src}
              alt={`${COMPANY_NAME} product showcase`}
              className="w-full h-auto object-cover"
              loading="lazy"
            />
          </div>
        </div>
      </div>
    </div>
  );
};

export default AboutUs;
