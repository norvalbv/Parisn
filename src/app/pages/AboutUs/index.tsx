import React, { ReactElement } from 'react';
import CardWrapper from 'components/CardWrapper';

const AboutUs = (): ReactElement => {
  return (
    <CardWrapper cardType="centered" className="w-1/2">
      <p className="text-center">
        {' '}
        This is a DEMO SITE. It is not currently responsive :( I also don&apos;t currently plan on
        making it mobile responsive.
        <span className="my-10 block">Check out my Github. @norvalbv</span>
        Site built with React, JavaScript, Typescript, Tailwind, Formik, AWS, Azure Devops, and a
        bunch of other stuff!
      </p>
    </CardWrapper>
  );
};

export default AboutUs;
