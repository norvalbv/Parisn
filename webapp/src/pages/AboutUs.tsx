import React from "react";

const AboutUs = () => {
  return (
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ex iste ratione
      beatae hic saepe provident temporibus, id repellendus. Veniam ratione illo
      consectetur cupiditate quisquam eaque delectus ipsa assumenda sed dolorum
      et, numquam nihil voluptas aut corrupti explicabo molestiae, doloribus
      aperiam placeat quasi debitis nemo. Modi inventore, vel commodi voluptate
      accusantium aliquam fugit ipsam illo unde. Quasi vero quo similique ex
      maiores commodi unde, voluptas distinctio corrupti iste ducimus laboriosam
      saepe impedit dicta dolorum, soluta quae, harum consectetur! Nostrum fugit
      vero sequi quam, corporis unde libero dicta, itaque deserunt iure soluta
      nihil quibusdam excepturi suscipit repellat, placeat hic. Recusandae,
      voluptatem beatae!
    </p>
  );
};

export default AboutUs;
