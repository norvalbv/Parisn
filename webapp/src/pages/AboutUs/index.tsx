import React, { ReactElement } from 'react';

const AboutUs = (): ReactElement => {
  return (
    <div className="h-screen grid place-items-center text-center">
      <p>
        This is a DEMO SITE. It is not currently responsive :( I also don&apos;t currently plan on
        making it mobile responsive.
        <span className="block my-10">Check out my Github. @norvalbv</span>
        Site built with React, JavaScript, Typescript, Tailwind, Formik, DaisyUI, AWS, Azure Devops,
        and a bunch of other stuff!
      </p>
    </div>
  );
};

export default AboutUs;
