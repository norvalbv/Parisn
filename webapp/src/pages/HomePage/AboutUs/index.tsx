import Radial from 'components/SVG/Design';
import React, { ReactElement } from 'react';
import PersonHoldingPhone from 'TempSVGImages/PersonHoldingPhone';

const AboutUs = (): ReactElement => {
  return (
    <div className="flex relative items-center w-1/2 mx-auto gap-8">
      <Radial />
      <div>
        <span className="mb-8 inline-block uppercase">Parisn.com</span>
        <p>
          Launched in 2021 allows you to choose the price you pay for your items in a reverse
          bidding strategy technique. Starting at £1,000 per item, over time the price of the item
          drops to £0.
        </p>
      </div>
      <PersonHoldingPhone />
    </div>
  );
};

export default AboutUs;
