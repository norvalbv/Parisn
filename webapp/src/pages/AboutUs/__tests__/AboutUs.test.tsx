import React from 'react';
import renderer from 'react-test-renderer';

import AboutUs from '..';

describe('<AboutUs />', () => {
  test('AboutUs props with mandatory props', () => {
    const tree = renderer.create(<AboutUs />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
