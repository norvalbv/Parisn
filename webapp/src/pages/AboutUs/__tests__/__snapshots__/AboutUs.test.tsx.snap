// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutUs /> AboutUs props with mandatory props 1`] = `
<p>
  Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ex iste ratione beatae hic saepe provident temporibus, id repellendus. Veniam ratione illo consectetur cupiditate quisquam eaque delectus ipsa assumenda sed dolorum et, numquam nihil voluptas aut corrupti explicabo molestiae, doloribus aperiam placeat quasi debitis nemo. Modi inventore, vel commodi voluptate accusantium aliquam fugit ipsam illo unde. Quasi vero quo similique ex maiores commodi unde, voluptas distinctio corrupti iste ducimus laboriosam saepe impedit dicta dolorum, soluta quae, harum consectetur! Nostrum fugit vero sequi quam, corporis unde libero dicta, itaque deserunt iure soluta nihil quibusdam excepturi suscipit repellat, placeat hic. Recusandae, voluptatem beatae!
</p>
`;
